### ISSUE TITLE
you said about improve my AL account so would you let me know more detail about that?

### ISSUE THREAD

**Issue by atherdon** [atherdon/how-to-sale#2]
Yes, for the first time it's easy. Just compare your upwork account - how much stuff there you have. and how limited information did you put in AL profile.

do you want more details or this is clear thing to you?

**Comment by whitesnow9291** [comment]
I will follow all the steps to you.
So please let me know what I should do.

**Comment by atherdon** [comment]
ok, first of all - you need to 

- put more information.
- more links
- for sure you need to add your projects.

you can grab my account and see what sections that i put. but you can put more information - it's upon to you.

my goal is to prepare your account and then we'll apply into few job positions that AL have.
feel free to ask more questions if something is not clear

**Comment by whitesnow9291** [comment]
Yes, I will, 
What is the next step?

**Comment by atherdon** [comment]
next step is show me updates. i'll review and maybe will make adjustments.
and put here link to your AL account. it'll save some time

**Comment by whitesnow9291** [comment]
OK.

**Comment by whitesnow9291** [comment]
I have done many projects in Upwork, but I worked with individual clients, so I can't remember their company and not sure if there are member of angel list,
How can I fill those fields?

**Comment by atherdon** [comment]
it's a bad news. 
it's not necessary to link with your clients here.
i want to have a list of projects - and this will show how much work you've done before. i assume you have at least 5 projects(maybe 10 - its better) that you want to share with your future bosses

**Comment by whitesnow9291** [comment]
Sure, I have done more than 5 projects.

**Comment by atherdon** [comment]
ok, soon this will know more people :)

**Comment by whitesnow9291** [comment]
I have done profile addition.

**Comment by atherdon** [comment]
> and put here link to your AL account. it'll save some time

**Comment by whitesnow9291** [comment]
Yes, I have added AL account linke in my bio text, because there is no setting for AL link in profile setting page.

**Comment by atherdon** [comment]
i mean put it here. on comments. so i don't need to search it

**Comment by atherdon** [comment]
add github account to AL profile too. FB link too.
and add that you open to networking and looking for a job. i think there two checkboxes in your profile settings

**Comment by atherdon** [comment]
![arthur tkachenko angellist](https://user-images.githubusercontent.com/1469198/39451504-89fd0f9c-4cd7-11e8-8cf1-6475d05e8e4b.png)

**Comment by atherdon** [comment]
check all that things

**Comment by whitesnow9291** [comment]
Yes, this is Angel account.
https://angel.co/nomura-nori
Yes, I will add FB and check all checkboxes.

**Comment by atherdon** [comment]
i assume it'll require to fill additional form in order to make your acc
more searchable for jobs..
and we'll be able to start to applying at AL for a few companies. we can do
it in a different ways.
we can try to use my network. - so i'll try to sell you :)
or you can apply by self. but i'll require some sort of tracking. i mean i
want to know to which companies you applying, what text you send to them,
etc, etc, etc...


On 30-Apr-2018 at 09:42 PM, Nomura Nori wrote:


Yes, this is Angel account.
https://angel.co/nomura-nori
Yes, I will add FB and check all checkboxes.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/atherdon/how-to-sale/issues/2#issuecomment-385537842>,
or mute the thread

**Comment by whitesnow9291** [comment]
How can I use your net work?

**Comment by atherdon** [comment]
well i have 3k mutual connections and 60k second degree of connections. if
they hiring = maybe i can intro you. maybe this will increase your chances.
but at first we'll need to find that companies, master how to use hubspot
crm and create an intro message - like for linkedin we need to have :)


On 30-Apr-2018 at 10:20 PM, Nomura Nori wrote:


How can I use your net work?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/atherdon/how-to-sale/issues/2#issuecomment-385546198>,
or mute the thread

**Comment by whitesnow9291** [comment]
Ok, great.

**Comment by whitesnow9291** [comment]
What is the next step I should do?
